README.md

```diff
@@ -1,2 +1,70 @@
-# relativepath.tech
-The RelativePath Website
+# React + TypeScript + Vite
+
+Template from [here](https://github.com/tailwindtoolbox/Landing-Page/blob/master/index.html).
+
+## Requirements
+
+For our developer setup we have assumed Ubuntu 22.04 as our base OS.
+
+## Description
+
+A simple setup using:
+
+- Vite ( build tool)
+- TypeScript (build tool)
+- React (UI)
+- Tailwind CSS (UI)
+- Example of the project file structure
+
+## Setup
+
+### Automatic (Recommended)
+
+1. Curl the `setup.sh` script and run it with bash on your machine:
```

Comment:
Let's assume we're running this all from the repo, instead of checking out the repo in the scripts. So you'd have the assumption that "we already are in the repo" which is the same assumption we'll have when running the scripts from the pipeline.

Reply:
This has been addressed.